src/frontend/Komponenter/Personoversikt/Infotrygdperioderoversikt.tsx

```diff
@@ -98,11 +115,24 @@ export const Infotrygdperioderoversikt: React.FC<{
     const infotrygdPerioder = useDataHenter<InfotrygdPerioderResponse, null>(
         infotrygdPerioderConfig
     );
+    const historiskPensjonConfig: AxiosRequestConfig = useMemo(
+        () => ({
+            method: 'GET',
+            url: `/familie-ef-sak/api/historiskpensjon/${personIdent}`,
+        }),
+        [personIdent]
+    );
+
+    const historiskPensjon = useDataHenter<IHistoriskPensjon, null>(historiskPensjonConfig);
```

Comment:
Burde historiskPensjon kunne være en helt egen komponent som selv er ansvarlig for å hente data? 
Og som då brukes i `InfotrygdEllerSummertePerioder`?

Reply:
Eneste fordelen her er at vi gjør kallen samtidig?

Reply:
Flyttet ut